recipes/xerces-c/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version="3.1.3" %}
+
+package:
+  name: xerces-c
+  version: {{ version }}
+
+source:
+  url: http://mirrors.sonic.net/apache//xerces/c/3/sources/xerces-c-{{version}}.tar.gz # [not win]
+  fn: xerces-c-{{version}}.tar.gz # [not win]
+  md5: 70320ab0e3269e47d978a6ca0c0e1e2d # [not win]
+
+  url: http://mirrors.sonic.net/apache//xerces/c/3/sources/xerces-c-{{version}}.zip # [win]
+  fn: xerces-c-{{version}}.zip # [win]
+  md5: 4b3ab521a3854c774b267005fb4d8ebd # [win]
```

Comment:
Usually, the difference is windows line endings.  However, they don't make any distinction between the different archives.  I think they are the same archive in 4 different formats.